wicket-core/src/main/java/org/apache/wicket/protocol/http/WebApplication.java

```diff
@@ -1111,4 +1115,14 @@ public ContentSecurityPolicySettings getCspSettings()
 		}
 		return cspSettings;
 	}
+
+	public void enableCoop(CoopConfiguration coopConfig)
```

Comment:
I don't think we should add these helpers, WebApplication has enough methods already and adding a listener quite easy.

Reply:
How about this idea:
- add the CoopConfiguration and CoepConfiguration as fields in SecuritySettings.
- add third mode `DISABLED` to the existing ones (`ENFORCING` and `REPORTING`)
- in WebApplication#init() add some logic to auto-add the Coop/Coep listener(s) when they are enabled

This way the developer will have to configure the security settings and don't bother how they are applied.